snappi_ixnetwork/ixnetworkapi.py

```diff
@@ -74,9 +74,15 @@ def snappi_config(self):
         return self._config
 
     def get_config_object(self, name):
+        if self._config_objects.get(name):
+            raise NameError("{0} is not avialable any Device object"
+                            .format(name))
         return self._config_objects[name]
 
     def get_device_encap(self, name):
+        if self._device_encap.get(name) is None:
+            raise NameError("{0} is not avialable any Device object"
```

Comment:
The error message should be `"snappi object named {0} not found".format(name)`. Feel free to rephrase it, but we should use similar error message in get_config_object and any other func that looks for obj based on keys.

Reply:
modify error message